packages/@sanity/preview/src/components/SanityDefaultPreview.js

```diff
@@ -79,6 +79,17 @@ export default class SanityDefaultPreview extends React.PureComponent {
     }
   }
 
+  renderIcon = options => {
+    const {type} = this.props
+    const Icon = type.icon
+    if (Icon) {
```

Comment:
You can simplify this with `return Icon && <Icon />`. The comment should be removed too.

Reply:
👍